<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaemoDeviceConfigWizardStartPage</class>
 <widget class="QWizardPage" name="MaemoDeviceConfigWizardStartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>The name to identify this configuration:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="maemoVersionLabel">
     <property name="text">
      <string>The system running on the device:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="fremantleButton">
       <property name="text">
        <string>Maemo 5 (Fremantle)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="harmattanButton">
       <property name="text">
        <string>Maemo 6 (Harmattan)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="meegoButton">
       <property name="text">
        <string>Meego</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>The kind of device:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="qemuButton">
       <property name="text">
        <string>Emulator (Qemu)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hwButton">
       <property name="text">
        <string>Hardware Device</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hostNameLabel">
     <property name="text">
      <string>The device's host name or IP address:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="hostNameLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
